### PR TITLE
Adding foreman_kubevirt Plugin

### PR DESCRIPTION
**Pull request by masayag**

### CODE REVIEW

**Comment by masayag** [comment]
I guess that this one will fail until https://github.com/theforeman/foreman-packaging/pull/3647 is merged

File: manifests/plugin/foreman_kubevirt.pp [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Installs foreman_kubevirt plugin
+class foreman::plugin::foreman_kubevirt {
+  foreman::plugin {'foreman_kubevirt':
+  }
+}
```
Comment (ekohl) [inline manifests/plugin/foreman_kubevirt.pp]: Nit: could you add newlines?

File: manifests/plugin/foreman_kubevirt.pp [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Installs foreman_kubevirt plugin
+class foreman::plugin::foreman_kubevirt {
+  foreman::plugin {'foreman_kubevirt':
```
Comment (ekohl) [inline manifests/plugin/foreman_kubevirt.pp]: The plugin already prepends `foreman_` by default:

```suggestion
  foreman::plugin { 'kubevirt':
```

File: manifests/plugin/foreman_kubevirt.pp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Installs foreman_kubevirt plugin
+class foreman::plugin::foreman_kubevirt {
```
Comment (ekohl) [inline manifests/plugin/foreman_kubevirt.pp]: Normally we don't include the `foreman_` prefix:
```suggestion
class foreman::plugin::kubevirt {
```

**Comment by masayag** [comment]
@ekohl Is there anything else needed for this PR ?

**Pull request merged**